dynamorestart/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM composer:2
```

Comment:
I don't know how similar these things are to idp-id-broker. If it's nearly the same, then we should probably extract it to a separate repo that can be shared. Obviously a separate task, so if it's possible, let's add a task card for it.